docs/learning/Display-Comparison.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Display Comparison
+nav_exclude: false
+has_children: false
+parent: Learning
+search_exclude: false
+last_modified_date: 2023-02-04
+---
+
+# Display Comparison
+
+## Why is this important?
+Displays are not exactly the same, and different display types have different pros and cons.
+
+## Panel Types:
+
+### Twisted Nematic (TN)
+they are the most common type of LCD panels that is still used and manufactuered today. They offer fast response times and high refresh rates, yet they have limited viewing angles and poor colors.
+
+### Vertical Alignment (VA)
+Has more saturated colors and higher contrast ratio that TN panels, making them a good choice for watching movies and other multimedia content. But they have the slowest responce times between the 3 types of panels.
+
+### In-Plane Switching (IPS)
+IPS is the newset technology between the 3. They offer the best colors reproduction, having a wide viewing angels and relatively low response times. However they are more expensive than the other two and can suffer from backlight bleeding defect. 
+
+### Main similarities between TVs and monitors:
+Both TV and monitor are electronic display devices that show images and videos, they share common video input ports like HDMI and VGA. They also came with same panel types such as IPS, TN, and VA.
+
+### Main Differences
+
+|  | Monitor | TV |
+| ---- | ---- | ---- |
+| Size | Typically smaller, with sizes ranging from 19 inches to 50 inches. Laptop screens range from 11 inches to 17 inches. | Generally larger, ranging from 30 inches to over 80 inches. |
+| PPI (pixels per inch) | Typically have higher density. | Usually have lower density. |
+| Refresh Rate | Comes with higher rates such as 60, 144, 165, 240 and 360Hz | Commonly between 60-165Hz. |
+| Input Ports | Fewer ports than TVs, being HDMI and DP (modern) or VGA and DVI (legacy) | More ports including HDMI, VGA, RCA connectors, YPbPr component video, DIN connector, S-Video, and RF connector (Coaxial connector). |
+
+Due to the various factors mentioned above, Monitors are preferred over TVs for tasks that require high levels of detail, like photo and video editing, or fast-paced activities, like gaming.
+
+## Issues When Using a TV with a Computer
+### Over/Underscan
+When you use a TV as a monitor for pc, you may notice the edges of the display are hidden beyond the borders of the screen, as if the picture is zoomed in. That is because of **overscan**. Or you may see black bars serrounding it, which is **underscan**. This can usually solved by adjusting the TV’s display settings,  or changing resolution/display scaling in the computer settings or GPU control panel.
```

Comment:
When you use a TV as a monitor for PC, you may notice the edges of the display are hidden beyond the borders of the screen, as if the picture is zoomed in. That is because of **overscan**. The opposite of which, where you see black bars surrounding it is called **underscan**. This can usually solved by adjusting the TV’s display settings,  or changing resolution/display scaling in the computer settings or GPU control panel.